src/Microsoft.Health.Dicom.Api/Controllers/WorkitemController.Add.cs

```diff
@@ -47,9 +48,11 @@ public partial class WorkitemController
         [VersionedRoute(KnownRoutes.CreateWorkitemInstancesRoute, Name = KnownRouteNames.VersionedWorkitemInstance)]
         [Route(KnownRoutes.CreateWorkitemInstancesRoute, Name = KnownRouteNames.WorkitemInstance)]
         [AuditEventType(AuditEventSubType.AddWorkitem)]
-        public async Task<IActionResult> AddAsync(string workitemInstanceUid = null)
+        public async Task<IActionResult> AddAsync()
         {
-            return await PostAsync(workitemInstanceUid);
+            // The Workitem UID is passed as the name of the first query parameter 
+            string workitemUid = HttpContext.Request.Query.Keys.FirstOrDefault();
```

Comment:
Can't we do [FromQuery] workitemInstanceUID?

Reply:
Can't because we inspect the parameter key, not the value.